src/transform.ts

```diff
@@ -68,6 +68,13 @@ export async function transform(
         query = `?${query}`
 
       const resolvePaths = (file: string) => {
+        if (!dir) {
+          let filePath = relative(root, file)
+          if (!filePath.startsWith('.'))
+            filePath = `/${filePath}`
+          return { filePath, importPath: filePath }
+        }
```

Comment:
I guess maybe we should require a virtual module to always glob from an absolute path